handlers/line/line.go

```diff
@@ -1,3 +1,178 @@
 package line
 
-/* no logs */
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"github.com/nyaruka/courier/utils"
+	"github.com/nyaruka/gocommon/urns"
+	"net/http"
+	"time"
+
+	"github.com/nyaruka/courier"
+	"github.com/nyaruka/courier/handlers"
+)
+
+var sendURL = "https://api.line.me/v2/bot/message/push"
+var maxMsgLength = 1600
```

Comment:
Do we know what this is exactly from their API docs?

Reply:
I see it is 2000 on the docs, I update that